kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml

```diff
@@ -55,6 +55,19 @@ productionModeEnabled: true
 # The name must be lowercase
 weblogicCredentialsSecretName: domain1-weblogic-credentials
 
+# Name of the Kubernetes secret to access the Docker Store to pull the WebLogic Server Docker image
+# The presence of the secret will be validated when this parameter is enabled.
+#imagePullSecretName:
```

Comment:
imagePullSecretName already exists in the inputs file, see line #49.

Reply:
Huh.  Maybe this came from a bad merge?  Fixed.